docs/cse/records-signals-entities-insights/configure-custom-insight.md

```diff
@@ -36,10 +37,10 @@ There are two ways you can define a Custom Insight. You can specify that the Ins
 * One or more selected rules fire a Signal.
 * Signals whose name matches a specified wildcard expression are fired. 
 
-Which method should you use? The difference is whether you’re going to create an Insight based on the name of the rule that fired the Signal, or based on the name of the Signal that was fired. Typically, Signals that a rule generates have the same name as the Signal. That is not the case with CSE’s normalized rules. That’s because normalized rules, for example [Normalized Threat rules](../rules/normalized-threat-rules.md), are written to work with multiple data sources. The names of the Signals that a normalized rule fires vary by data source. So, if you want your
+Which method should you use? The difference is whether you’re going to create an Insight based on the name of the rule that fired the Signal, or based on the name of the Signal that was fired. Typically, Signals that a rule generates have the same name as the Signal. That is not the case with CSE’s normalized rules. That’s because normalized rules, for example [Normalized Threat rules](/docs/cse/rules/normalized-threat-rules/), are written to work with multiple data sources. The names of the Signals that a normalized rule fires vary by data source. So, if you want your
 Custom Insight configuration to generate Insights for Signals fired by normalized rules, you should base it on Signal names, rather than rule names.
 
-When the conditions of a Custom Insight configuration are met during the currently configured [detection window](set-insight-generation-window-threshold.md), an Insight will be generated for each Entity involved. In other words, if each of the Signals in a Custom Insight configuration fired on a different Entity, an Insight will be created on each of those Entities. The generated Insights will include not only the Signals that it fired on, but also any related Signals. 
+When the conditions of a Custom Insight configuration are met during the currently configured [detection window](/docs/cse/records-signals-entities-insights/set-insight-generation-window-threshold/), an Insight will be generated for each Entity involved. In other words, if each of the Signals in a Custom Insight configuration fired on a different Entity, an Insight will be created on each of those Entities. The generated Insights will include not only the Signals that it fired on, but also any related Signals. 
 
 This example Custom Insight configuration will generate an Insight as a result of the **McAfee ePO - Virus Outbreak** rule firing a Signal. 
```

Comment:
the image underneath this doesn't appear to match the text

Reply:
Fixed